tests/types/scripts/test.mjs

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env zx
+
+import fs from 'fs';
+import 'zx/globals';
+
+const VERSIONS = [
+  'next',
+  'beta',
+  'latest',
+  '3.9.7',
+  '3.8.3',
+  '3.7.4',
+  '3.6.3',
+  '3.5.1',
+  '3.4.4',
+  '3.3.3',
+  '3.2.1',
+  // '3.1.1',
```

Comment:
either remove or comment why it is commented out.

Reply:
👍 updated